include/domain_bridge/qos_options.hpp

```diff
@@ -87,7 +87,7 @@ class QosOptions
    * If nullopt_t is returned, then use automatic matching.
    */
   DOMAIN_BRIDGE_PUBLIC
-  std::optional<std::int64_t>
```

Comment:
Should we also clean up ``std::`` in the function declarations above ? Like in ``depth(const std::size_t & depth);``

Reply:
yes

Reply:
Fixed in https://github.com/ros2/domain_bridge/pull/73/commits/4e66a4fad7e145e9beae59831c1b66c9d3869ef7